src/device/libpixy_m4/src/camera.cpp

```diff
@@ -601,22 +601,22 @@ void cam_loadParams()
 	prm_setShadowCallback("Camera Brightness", (ShadowCallback)cam_shadowCallback);
 
 	prm_add("Auto Exposure Correction", PRM_FLAG_ADVANCED | PRM_FLAG_CHECKBOX, 
-		"@c Camera Enables/disables Auto Exposure Correction. (default enabled)", UINT8(1), END);
+		"@c Camera Enables/disables Auto Exposure Correction. (default disabled)", UINT8(0), END);

```

Comment:
I'm just curious.... does the string actually do anything (i.e. is it parsed to actually process a command) or is it just displayed somewhere?

Reply:
So is it safe to say that 1 is enabled and 0 is disabled?
If so, could we perhaps use a #define for clarity?

Reply:
I guess it's ok the way it is since the ascii string is there.  The only reason I was able to conclude enable/disable was because of the ascii string.